docs/installation.rst

```diff
@@ -6,13 +6,14 @@ Installation
 
 Stable release
 --------------
-To install xclim via pip, run this command in your terminal:
+
+To install `xclim` via `pip`, run this command in your terminal:
 
 .. code-block:: shell
 
     $ pip install xclim
 
-This is the preferred method to install xclim, as it will always install the most recent stable release.
+This is the preferred method to install `xclim`, as it will always install the most recent stable release.
```

Comment:
Is it ?
I know this is not related to the PR, but it seems to me that the preferred method to install xclim would rather be `conda` !

Reply:
I suppose so. This might be an artifact from before we were on PyPI (or were newly arrived at PyPI). It's probably fine to simply suggest PyPI or Anaconda.